ocptv/output/config.py

```diff
@@ -68,22 +70,44 @@ def enable_runtime_checks(self, enable: bool):
         with self._lock:
             self._enable_runtime_checks = enable
 
+    @property
+    def timezone(self) -> ty.Union[tzinfo, None]:
+        with self._lock:
+            return self._tzinfo
+
+    @timezone.setter
+    def timezone(self, tzinfo: ty.Union[tzinfo, None]):
+        with self._lock:
+            self._tzinfo = tzinfo
+
 
 # module scoped configuration (similar to python logging)
 _config: Config = Config()
 
+if ty.TYPE_CHECKING:  # pragma: no cover
+    # We use a sentinel object that looks like a None for the static type checker.
+    # At runtime, we check against the specific instance of `object()` to make a decision
+    # on whether this parameter was passed in or not.
+    # This is useful for types where NoneType is actually a valid instance of the type.
+    _NOT_SET = None
+else:
+    _NOT_SET = object()
+
 
 @export_api
 def config(
     *,
     writer: ty.Optional[Writer] = None,
     enable_runtime_checks: ty.Optional[bool] = None,
+    timezone: ty.Union[tzinfo, None] = _NOT_SET,
 ):
     """
     Configure how the ocptv.output lib behaves.
 
     :param Writer writer: if provided, set the output channel writer.
     :param bool enable_runtime_checks: if provided, enables or disables runtime type checks.
+    :param tzinfo timezone: if provided, sets the timezone for the output formatted datetime fields.
+        To reset to local timezone, `None` value should be used.
```

Comment:
Spec specifies "The only guidance provided is that the implementation shall conform to this format, and should be referenced to UTC time when possible.".

Shouldn't the default therefore be UTC? Local timezone is non-deterministic in areas with summer time during the hour before and after the time gets moved back.

Reply:
moved everything to UTC; you are on point about the spec